string waypoint
topological_patroller/CheckPointAction[] action
